cloud/kubernetes/helm/templates/_helpers.tpl

```diff
@@ -45,7 +43,7 @@ volumeMounts:
 volumes:
 {{- end }}
 {{- range $i, $vName := tuple "data-store" "logs" "rms-shared" "rms-security" "rms-webstudio" }}
-{{- if or (and (eq $vName "data-store") (eq $.Values.bsType "sharednothing")) (and (eq $vName "logs") $.Values.persistence.logs) (and (eq $vName "rms-shared") (or $.Values.enableRMS $.Values.rmsDeployment)) (and (eq $vName "rms-security") $.Values.persistence.rmsSecurity $.Values.rmsDeployment) (and (eq $vName "rms-webstudio") $.Values.persistence.rmsWebstudio $.Values.rmsDeployment) }}
```

Comment:
`$.Values.persistence.rmsWebstudio $.Values.rmsDeployment)` is repeated twice and rms-security condition is also there

Reply:
Here `rms-security` is loop variable check. Condition repeats for `rms-webstudio` as well.